modules/images/webp-uploads/load.php

```diff
@@ -340,3 +340,112 @@ function webp_uploads_wp_get_missing_image_subsizes( $missing_sizes, $image_meta
 }
 
 add_filter( 'wp_get_missing_image_subsizes', 'webp_uploads_wp_get_missing_image_subsizes', 10, 3 );
+
+/**
+ * Filters on `the_content` to update the references for supported mime of images into the
+ * `webp_uploads_preferred_mime_type()` for the most part `image/webp` if the current
+ * attachment contains the targeted mime type.
+ *
+ * @since n.e.x.t
+ *
+ * @see the_content
```

Comment:
IMO not worth focusing on that filter, but what would be helpful is a pointer to the very related `wp_filter_content_tags()`.
```suggestion
 * @see wp_filter_content_tags()
```